database/index.js

```diff
@@ -0,0 +1,15 @@
+const { resolve } = require('path');
```

Comment:
Essa parada o próprio sequelize já resolve pra você. 
```
npx sequelize init
```

E ele já vai lidar com todas as configs de envs e tals, eu acho mais simples e less error prone.

Reply:
Então marcle eu tentei usar o código desse sequelize init e fui alterando ele, acabei chegando no mesmo arquivo de configuração que eu fiz kk, poderia me mostrar como tu faria ?

Reply:
@antoniolopesg de inicio eu usaria a mesma configuração do sequelize, acho que a arquitetura ela tem que emergir da evolução do sistema. No começo a gente segue alguns padrões. Então eu faria isso, seguiria o jeito normal de definir models. Ou perder muito tempo criando um arquivo de configuração para importar e de define nos models on the fly, não gosto de perder tempo. :smile:

Reply:
Mas tipo assim, esse teu arquivo tá super de boas. Não precisa mexer nele.